app/services/push_notifications/event_notifier/post_logic.rb

```diff
@@ -0,0 +1,24 @@
+module PushNotifications
+  class EventNotifier
```

Comment:
I don't follow why we're nesting this `PostLogic` here? what's the actual reasoning? What confuses me I think is the fact that `EventNotifier` is both a class and a namespace 🤔

Reply:
That's correct, I could have put `PostLogic` calss even in the same file where `EventNotifier` is defined... but I don't like it much. `PostLogic` basically is a local business domain, only used inside `EventNotifier`... at least for now...